clang/lib/Sema/CheckedCAnalysesPrepass.cpp

```diff
@@ -126,44 +169,59 @@ class PrepassHelper : public RecursiveASTVisitor<PrepassHelper> {
     }
 
     void DumpBoundsVars(FunctionDecl *FD) {
+      PrintDeclMap<const VarDecl *>(FD, "BoundsVars Lower",
+                                    Info.BoundsVarsLower);
+      PrintDeclMap<const VarDecl *>(FD, "BoundsVars Upper",
+                                    Info.BoundsVarsUpper);
+    }
+    // Print a map from a key of type T to a set of elements of type T,
+    // where T should inherit from NamedDecl.
+    // This method can be used to print the BoundsVars and
```

Comment:
Nit: I suggest updating this comment to mention `BoundsVarsUpper` and `BoundsVarsLower` rather than `BoundsVars` and `BoundsSiblingFields`.